### PR TITLE
3.0

### PR DESCRIPTION
**Pull request by pulse14**
Translation, minor changes
Error in an ORM example

### CODE REVIEW

File: en/orm/entities.rst [hunk 1]
```diff
@@ -363,9 +363,9 @@ Lazy loading
     Lazy loading defers loading association data until it is absolutely
     required. While this can save CPU time because possibly unused data is not
     hydrated into objects, it can result in many more queries being emitted to
-    the database. For example looping over a set of articles & their comments
+    the database. **For example looping over a set of articles & their comments
     will frequently emit N queries where N is the number of articles being
-    iterated.
+    iterated.**
```
Comment (markstory) [inline en/orm/entities.rst]: I don't think this whole sentence needs to be bold.

File: fr/orm/saving-data.rst [hunk 2]
```diff
@@ -36,6 +36,10 @@ créer une nouvelle entity et de la passer à la méthode ``save()`` de la class
     // L'$article et l'entity $savedArticle retourné contient maintenant l'id
     $id = $savedArticle->id;
 
+En utilisant ``newEntity``, la méthode ``isNew()`` de l'entity nouvellement 
+crée renverra ``true``. C'est ainsi que, lors de l'appel de ``save()``, 
+CakePHP saura qu'il s'agit d'un ajout et non une mise à jour.
+
```
Comment (antograssiot) [inline fr/orm/saving-data.rst]: All this paragraph doesn't exis in the original en version and is already explain twice in this page.

File: fr/orm/saving-data.rst [hunk 3]
```diff
@@ -437,7 +443,7 @@ Par exemple, considérons le cas suivant::
             ['body' => 'A new comment'],
         ]
     ];
-    $articles->patchEntity($entity, $newData);
+    $articles->patchEntity($article, $newData);
     $articles->save($article);
```
Comment (antograssiot) [inline fr/orm/saving-data.rst]: changing $article here by $entity would also work.
Anyway, would you be able to update also the original example?

File: fr/orm/saving-data.rst [hunk 4]
```diff
@@ -429,7 +433,9 @@ Par exemple, considérons le cas suivant::
             ['body' => 'Second comment', 'id' => 2],
         ]
     ];
-    $entity = $articles->newEntity($data);
+    
+    $article = $articles->newEntity($data);
+    $articles->save($article);
```
Comment (antograssiot) [inline fr/orm/saving-data.rst]: I beleve that this is not needed.

**Comment by antograssiot** [comment]
@pulse14 appartement from my comments, translation work looks great thanks.

File: fr/orm/saving-data.rst (hunk 4 above)
Comment (pulse14) [inline fr/orm/saving-data.rst]: If we don't save, 'Second comment' with id 2 isn't created. And the next sentence make no sense.

**Pull request merged**